src/app/app.module.ts

```diff
@@ -1,18 +1,48 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
+// routing
+import { AppRoutingModule } from './app-routing.module';
 
+// components
 import { AppComponent } from './app.component';
+import { NotFoundComponent } from './not-found/not-found.component';
+
+// services
+import { UserService } from './core/services/user.service';
+
+// @ngrx
+import { StoreRouterConnectingModule } from '@ngrx/router-store';
+import { StoreModule } from '@ngrx/store';
+import { FormsModule } from '@angular/forms';
+
+// reducers
+import { reducers } from './store/app.reducers';
+import { EffectsModule } from '@ngrx/effects';
+import { UserEffects } from './users/store/users.effects';
+import {initialState} from "./store/app.state";
 
 
 @NgModule({
   declarations: [
-    AppComponent
+      AppComponent,
+      NotFoundComponent
   ],
   imports: [
-    BrowserModule
+      AppRoutingModule,
+      BrowserModule,
+      BrowserAnimationsModule,
+      FormsModule,
+      StoreRouterConnectingModule,
+      StoreModule.forRoot(reducers, {
+          initialState: initialState
+      }),
+      EffectsModule.forRoot([
+          UserEffects
```

Comment:
because your `user.module` is loaded lazily you can add `UserEffects` inside it. Simply use `EffectsModule.forFeature` inside `user.module`